src/clients/ntp-client_proposal.rb

```diff
@@ -210,21 +210,33 @@ def ui_init(rp, first_time)
       rp = deep_copy(rp)
       cont = VBox(
         VSpacing(0.5),
+        HBox(
+          HSpacing(3),
+          HWeight(
+            1,
+            Left(
+              ComboBox(
+                Id(:ntp_address),
+                Opt(:editable, :hstretch),
+                # combo box label
+                _("&NTP Server Address")
+              )
+            )
+          ),
+          HWeight(
+            1,
+            VBox(
+              VSpacing(1),
```

Comment:
The code below uses `Label("")` trick to have an empty space. This should be consistent.

Reply:
That trick does not work well with `TextMode`, so at least will add the conditional.